cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/util/CdcsdkContainer.java

```diff
@@ -226,4 +265,20 @@ public GenericContainer<?> buildForS3Sink() throws Exception {
 
         return cdcsdkContainer;
     }
+
+    public GenericContainer<?> buildForPubSubSink() throws Exception {
```

Comment:
Is it possible to consolidate all the 3 functions `buildForS3Sink` , `buildForKafka` and this one? The only difference seems to be in setting env. Can it be replaced by a `build(Map<> env)` fn? cc @vaibhav-yb

Reply:
Done